index.html

```diff
@@ -1,14 +1,158 @@
 <!DOCTYPE html>
```

Comment:
- [ ] Please, consider that there is no footer section in the mobile version in `index.html`
![2022-04-12 10 26 12](https://user-images.githubusercontent.com/49064106/162894577-f856d084-3bda-45fc-a099-69c190066b6b.gif)

Reply:
@elmar8287 Hi there, yes it should not have a footer in the home page. Footer will be shown only in the about page.

Reply:
> @elmar8287 Hi there, yes it should not have a footer in the home page mobile version. Footer will be shown only in the about page mobile version. https://www.behance.net/gallery/29845175/CC-Global-Summit-2015

@mavericks-db Sorry for the misunderstanding, you have not created the footer for the Home page.

Reply:
@elmar8287 Yes, I added already. Thanks 👍